aws_cloudtrail_rules/aws_root_failed_console_login.yml

```diff
@@ -13,7 +13,7 @@ Reports:
   MITRE ATT&CK:
     - Initial Access:Valid Accounts
 Description: >
-  Someone attempted to log into the root account from the AWS console and failed. This could be an indication of a password guessing attack against your root account.
+  Deprecated. Please see AWS.Console.LoginFailed instead.
```

Comment:
should the comment here be `AWS.Console.RootLoginFailed`?

Reply:
Oh yeah, good catch.